extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java

```diff
@@ -100,7 +100,7 @@ private String guessDriver(Optional<DataSourceDriverBuildItem> dataSourceDriver)
             return HSQLDBDelegate.class.getName();
         }
 
-        if (resolvedDriver.contains("com.microsoft.sqlserver.jdbc.SQLServerResource")) {
+        if (resolvedDriver.contains("com.microsoft.sqlserver.jdbc")) {
```

Comment:
Good catch. Maybe we can make it `com.microsoft.sqlserver.jdbc.SQLServerDriver` instead?

Reply:
Indeed you're right.
Will do

Reply:
Thanks. Can we squash the two commits into one, please! 
Once that is done, this PR should be good to be merged.